.github/workflows/build.yaml

```diff
@@ -35,25 +39,20 @@ jobs:
         uses: ASzc/change-string-case-action@v1
         with:
           string: ${{ github.repository }}
+      
+      - name: Convert the GitHub repository Owner name to lowercase
+        id: repository-owner
+        uses: ASzc/change-string-case-action@v1
+        with:
+          string: ${{ github.repository_owner	 }}
 
       - name: Set outputs
         id: cache
         run: |
-          # Set variables according to version of ubuntu
-          if [[ "${{github.base_ref}}" == "master" || "${{github.ref}}" == "refs/heads/master" ]]; then
-            echo "::set-output name=CACHE_KEY_BUILD::${{ hashFiles('.github/workflows/build/Dockerfile.ubuntu-1604') }}"
-            echo "::set-output name=CACHE_KEY_LINT::${{ hashFiles('.github/workflows/lint/Dockerfile.ubuntu-1604') }}"
-            echo "::set-output name=UBUNTU_VERSION::ubuntu-1604"
-            echo "::set-output name=distribution::xenial"
-          fi
-          if [[ "${{github.base_ref}}" == "ubuntu-20.04-upgrade" || "${{github.ref}}" == "refs/heads/ubuntu-20.04-upgrade" ]]; then
-            echo "::set-output name=CACHE_KEY_BUILD::${{ hashFiles('.github/workflows/build/Dockerfile.ubuntu-2004') }}"
-            echo "::set-output name=CACHE_KEY_LINT::${{ hashFiles('.github/workflows/lint/Dockerfile.ubuntu-2004') }}"
-            echo "::set-output name=UBUNTU_VERSION::ubuntu-2004"
-            echo "::set-output name=distribution::focal"
-          fi
-     
-
+          echo "::set-output name=UBUNTU_VERSION::ubuntu-2004"
+          echo "::set-output name=distribution::focal"
+          echo "::set-output name=shortSHA::sha-$(git rev-parse --short HEAD)"
```

Comment:
@pSchlarb, @udosson Have we made a conscious decision to do away with the support for ubuntu 16.04 builds on this branch?  Would the plan then be to only support 16.04 builds on the main branch and 20.04 on this branch until such time as this branch is merged?  If so we should do away with all of the other 16.04 related CI/CD sources on this branch.

Reply:
Would be in favor for that.